download-references.nf

```diff
@@ -173,14 +189,20 @@ process download_ericscript {
     """
     wget -N https://raw.githubusercontent.com/circulosmeos/gdown.pl/dfd6dc910a38a42d550397bb5c2335be2c4bcf54/gdown.pl
     chmod +x gdown.pl
```

Comment:
Let's ditch this and use [conda package](https://anaconda.org/conda-forge/gdown) instead.
This would require one more change in `environment.yml`

```yaml
dependencies:
  - conda-forge::gdown=3.12.0
```